### DISCUSSION TITLE
Re-import zpool?

### DISCUSSION THREAD

**Discussion by Cronvs** [zbm-dev/zfsbootmenu#691] [Questions] [answered]
I am in the middle of setting a second distro in ZBM, as I understand from others you basically create a new dataset under the ROOT dataset, however, I have been reviewing the guides for setting up ZBM and trying to understand why we re-import the zpool during setup. Only difference I see is that it sets altroot, is this important for secondary distros, why should one do it?

**Comment by zdykstra** [comment, marked as answer]
It's just a convenient way to get the dataset mounted. If you want, create the dataset and set the mount point to /mnt and then mount it. Just remember to reset it back to /, with canmount set to noauto.
